modules/gapi/test/own/mat_tests.cpp

```diff
@@ -384,4 +384,32 @@ TEST(OwnMat, ROIView)
     << to_ocv(roi_view) << std::endl
     << expected_cv_mat  << std::endl;
 }
+
+TEST(OwnMat, CreateNegEmptyMat)
+{
+    // We use create method for empty own::Mat
+    Mat own_mat_t1, own_mat_t2, own_mat_t3;
+    EXPECT_THROW(own_mat_t1.create(cv::Size{-1, -1}, CV_8U), std::exception);
+    EXPECT_THROW(own_mat_t2.create(cv::Size{ 1, -1}, CV_8U), std::exception);
+    EXPECT_THROW(own_mat_t3.create(cv::Size{-1,  1}, CV_8U), std::exception);
+}
+
+TEST(OwnMat, CreateNegExistMat)
```

Comment:
Split this into small tests for specific cases instead of one large test with a bunch of checks
This way you will see that comments will be unnecessary

Reply:
+1

Reply:
@TolyaTalamanov I made this tests easier. Is split necessary?